test/k8sT/Updates.go

```diff
@@ -71,6 +71,9 @@ var _ = Describe("K8sUpdates", func() {
 		kubectl.Delete(l3Policy)
 		kubectl.Delete(demoPath)
 
+		// make sure that Kubedns is deleted correctly
```

Comment:
This isn't blocking, but this comment is fairly self-evident. What's not important is that we are deleting kube-dns, it is *why* we are deleting it. We need to make sure that we illustrate why we are performing certain operations at certain times, e.g. "delete kube-dns here because after upgrade tests for K8s 1.7, kube-dns goes into a bad state; see GH-XXXX for more context" - this illustrates we certain things are done instead of just saying what is evident from the code itself.